web/src/OverviewItemView.tsx

```diff
@@ -190,42 +194,28 @@ let RuntimeBoxStack = styled.div`
   // To truncate long resource names…
   min-width: 0; // Override default, so width can be less than content
 `
-
 let InnerRuntimeBox = styled.div`
   display: flex;
   align-items: center;
+  flex-grow: 1;
 `
-
-let OverviewItemActions = styled.div`
-  display: flex;
-  flex-direction: column;
-`
-
-let OverviewItemBuildBox = styled.div`
-  display: flex;
-  align-items: stretch;
-  flex-shrink: 1;
-  border-top: 1px solid ${Color.grayLighter};
-`
-
-let OverviewItemText = styled.div`
+let OverviewItemType = styled.div`
   display: flex;
   align-items: center;
-  white-space: nowrap;
```

Comment:
When I moved the Trigger + Trigger Mode buttons in the card layout, I checked text truncation again to make sure it was working properly. Along the way, I did a little refactoring of the styles related to various forms of text on the card (Resource Type, Resource Name, Build status).

I noticed some old CSS rules that made it look as though Type + Build text would be truncated. But it didn't actually work. But it kind of doesn't matter anyway because we control that text, and there's no option that's so long that it requires truncation. At least, I don't think so. (I didn't test all the variants extensively, I admit)

Anyway, since truncation wasn't working anyway, I made it more obvious that Resource Name is the only thing that's actively truncated, and tested it to make sure it works.